src/cli/index.ts

```diff
@@ -42,16 +43,33 @@ export default class Cli {
   private readonly argv: string[]
   private readonly cwd: string
   private readonly stdout: IFormatterStream
+  private readonly console: Console
 
-  constructor({ argv, cwd, stdout }) {
+  constructor({ argv, cwd, stdout, stderr }) {
     this.argv = argv
     this.cwd = cwd
     this.stdout = stdout
+    this.console = new Console(stderr)
```

Comment:
I was thinking we pass both stdout and stderr to this. 

```typescript
this.console = new Console({stdout, stderr})
```

It's working currently due to the constructor assuming we are passing only a stdout stream and error output goes to stdout if stderr is not supplied. We usually wouldn't print something to stdout using the console variable but I think its possible I doubt we would even send anything to stdout using the console variable we are creating but I'm unsure.

More info here:

https://nodejs.org/api/console.html#console_new_console_stdout_stderr_ignoreerrors

Reply:
Gotcha, fixed